### ISSUE TITLE
Timed out waiting for Namespace Creation

### ISSUE THREAD

**Issue by nikk0** [yugabyte/yugabyte-db#4938]
Hello all.

I am using YugabyteDB 2.1.8.2 running on a 3nodes cluster deployed with docker swarm(from the documentation).

The "create database" failed with  "Timed out waiting for Namespace Creation" and an inconsistency then appeared in the system : unable to delete this partially-created database.

```
docker@worker1:~$ docker exec -it 3d82c607c060 /home/yugabyte/bin/ysqlsh
ysqlsh (11.2-YB-2.1.8.2-b0)
Type "help" for help.

yugabyte=# create database nbe;
ERROR:  Timed out: Timed out waiting for Namespace Creation
yugabyte=# create database nbe;
ERROR:  Already present: Keyspace 'nbe' already exists
yugabyte=# drop database nbe;
ERROR:  database "nbe" does not exist
yugabyte=# create database nbe;
ERROR:  Already present: Keyspace 'nbe' already exists
yugabyte=# exit
docker@worker1:~$
```

Please help.

**Comment by nspiegelberg** [comment]
This issue is fixed as a part of #3979.  Although that task isn't complete, this specific issue is fixed in the next release (2.2)

**Comment by ddorian** [comment]
Closing because 2.2 is released: https://github.com/yugabyte/yugabyte-db/releases/tag/v2.2.0.0
